Automattic/wp-calypso
# Domains: Update domain mapping thank you page

#### Changes proposed in this Pull Request

In order to improve the user experience when buying domains and mappings, domain mapping instructions were updated in #50668. This PR updates the `Thank You` page, which is shown after a user buys a domain mapping, to reflect the same instructions that were changed in #50668.

Depending on the type of site the mapping was bought for - root domain or sub domain, for a simple or AT site - there should be one recommended setup (usually setting name servers) and one or two alternative, more advanced, options, usually setting A or CNAME records.

#### Testing instructions


Both simple and atomic sites should be tested, as well as root domain mappings and subdomain mappings.

1. In Calypso, go to Upgrades > Domains > Add a domain to this site
2. Select "Use a domain I own"
3. "Map your domain"
4. Insert a root domain or subdomain
5. Buy that domain mapping for your site
6. The "Thank you" page with further instructions to setup domain mapping will be shown.

Root domain mappings instructions differ if the site is simple or atomic.
- For simple sites: The recommended setup should be to set NS records pointing to our name servers, and the advanced setup should show the default IP addresses (*.24 and *.25) to be set as A records
- For AT sites: The recommended setup should be to set NS records pointing to our name servers, and the advanced setup should show the AT site's IP addresses to be set as A records

Subdomain mappings should show the following instructions: The recommended setup should be to set CNAME or NS records pointing to our name servers, and no advanced setup should be shown
